artemis-quorum-ri/src/test/java/org/apache/activemq/artemis/quorum/zookeeper/CuratorDistributedLockTest.java

```diff
@@ -337,21 +333,8 @@ public void beNotifiedOfAlreadyUnavailableManagerAfterAddingListener() throws Ex
       }
    }
 
-   private boolean ensembleHasLeader() {
-      return testingServer.getServers().stream().filter(CuratorDistributedLockTest::isLeader).count() != 0;
-   }
-
-   private static boolean isLeader(TestingZooKeeperServer server) {
-      if (server.getInstanceSpecs().size() == 1) {
-         return true;
-      }
-      long leaderId = server.getQuorumPeer().getLeaderId();
-      long id = server.getQuorumPeer().getId();
-      return id == leaderId;
-   }
-
    private void stopMajorityNotLeaderNodes(boolean fromLast) throws Exception {
-      List<TestingZooKeeperServer> followers = testingServer.getServers().stream().filter(Predicates.not(CuratorDistributedLockTest::isLeader)).collect(Collectors.toList());
+      List<TestingZooKeeperServer> followers = testingServer.getServers();
```

Comment:
Is this now just stopping 'a majority of nodes' rather than the original 'majority of non-leader nodes'?

Reply:
renamed

Reply:
I was not really pointing out the name wasnt descriptive, but that you appeared to have changed _what the method is trying to do_. As that could be a difference that alters expected behaviour for tests using it, it isnt immediately obvious as being desirable.

Reply:
Since leader api was removed there is no significant choice which instance stopped